proposals/proposal-template.md

```diff
@@ -0,0 +1,112 @@
+<!---
+Welcome! Thank you for contributing to the U.S. Web Design System.
+Your contributions are vital to our success and we are glad you want to share your ideas.
+
+You can find full instructions for creating a proposal in the uswds-proposals repo:
+https://github.com/amyleadem/uswds-proposals/edit/add-readme-and-template/README.md#submitting-a-new-proposal
+
+More information about contributing to USWDS can be found on the contribution page:
+https://designsystem.digital.gov/about/contribute/
+ -->
+
+## Summary*
+
+_Provide a short paragraph or bulleted list that gives an overview of the proposed component._
+
+## Motivation*
+
+_Explain why we should include this component in USWDS._
+
+- _What user need does this component address?_
+- _What problem does this component solve?_
+
+## Proposed solution*
+
+_Thoroughly describe the proposed component. Use the following sections to add detail:_
+
+### Functionality*
+
+_Describe the component's core functionality. If the component is dynamic, describe its interactive states._
+
+### Visuals*
+
+_Add diagrams, images, and/or wireframes that visually demonstrate this component._
+
+### Variants*
+
+_Describe any variants planned for this component._
+
+### Settings*
+
+_Describe which component elements should be customizable with theme settings._
+
+### Accessibility*
+
+_Identify any accessibility considerations._
+
+- _Will the component be compatible with assistive technologies?_
+<!-- 
+Consider the following:
+- Keyboard-only interaction
+- Screen readers
+- Voice command
+- High-contrast mode
+- Zoom magnification 
+-->
+- _Are there any audiences that might struggle with this component?_
+<!--Explain how this component will accommodate these needs.-->
+
+### Implementation plan*
+_Add details about how this plan should be implemented._
+
+- _Will the component be implemented in phases?_
+- _Are there any potential blockers?_
+- _Are there any dependencies?_
+
+### Testing plan*
+_Explain how to test this component._
+
+- _What kinds of tests will need to be performed?_
+- _Which elements or interactions will need to be tested?_
+- _Are there any specific audiences that should be included in user testing?_
+
+### Documentation*
+
+_Identify any documentation update requirements._
```

Comment:
@mejiaj Starting to wonder more and more if this documentation section is necessary. It made more sense to include it when the proposal process included other significant changes and not just components. 

From what I can see, every component update will need similar updates: a new documentation page and updated design files. I can't think of many scenarios where a contributor would (or should) identify additional documentation requirements. I am starting to wonder if it would just introduce confusion. 

Instead, identifying documentation requirements feels like something the design and dev team could identify during the [requirements phase](https://docs.google.com/document/d/1FGgG3SC9wKYMe1RvQe07xzgCWu-rxS-8y-xbNmjiZfM/edit#heading=h.ejxjih3e08y7). 
 
Curious to hear what the team thinks. What do you think about just removing this section for now?